src/azure-cli-core/azure/cli/core/tests/test_application.py

```diff
@@ -90,7 +90,7 @@ def test_expand_file_prefixed_files(self):
         f_with_bom.close()
 
         with open(f.name, 'w+') as stream:
-            stream.write('foo')
+            stream.write('foo \r\n')
```

Comment:
Is this intended to be a windows specific test?

Reply:
No, it's just intended to make sure that any of these characters are stripped from the end. The old logic only stripped off \n (which is why it didn't work on Windows)